src/main/reactjs/public/i18n/fi-FI/translation.json

```diff
@@ -55,6 +55,19 @@
           "buttons": {
             "add": "Lisää auktorisointi"
           },
+          "row": {
```

Comment:
Tää tuntuu mielestäni ylimääräiseltä nestaukselta. Pois?

Reply:
Nyt kun lisäsin myös julkaisuluvan muokkauksen niin jätänkö näin vai muokataanko jollain lailla?

Reply:
No mun mielestä tuo `row` on edelleen vähän ylimääräinen tässä. Jos tuntuu siltä että tarvitsee jotain ryhmittelyä, niin ehkä sitten voisi löytää jonkun kuvaavamman nimen.. `actions` tms?

Reply:
Vaihdan tilalle `actions`